gsrs-spring-boot-autoconfigure/src/main/java/gsrs/controller/AbstractImportSupportingGsrsEntityController.java

```diff
@@ -679,6 +679,34 @@ public ResponseEntity<Object> updateImport(@RequestBody JsonNode updatedJson,
         return new ResponseEntity<>(GsrsControllerUtil.enhanceWithView(itmd, queryParameters), HttpStatus.OK);
     }
 
+    @hasAdminRole
```

Comment:
We may need to adjust this for staging area and have some more configurable set of roles that can see the staging area. I currently think it may be best for super users or perhaps even users that are in a specific group ... maybe admins _or_ "staging area registrars"? Can we make this configurable somehow?

Reply:
This makes sense.